### PR TITLE
Check if duplicate IDs are used for Layer2NetworkConfiguration

### PR DESCRIPTION
**Pull request by chdxD1**

### CODE REVIEW

**Comment by Cellebyte** [comment]
should we not block that on apply?

File: pkg/reconciler/layer2.go [hunk 1]
```diff
@@ -69,6 +69,10 @@ func (r *reconcile) fetchLayer2(ctx context.Context) ([]networkv1alpha1.Layer2Ne
 		l2vnis = append(l2vnis, *item)
 	}
 
+	if err := r.checkL2Duplicates(l2vnis); err != nil {
```
Comment (Cellebyte) [inline pkg/reconciler/layer2.go:72]: should we not block mismatching resources before they are in cluster?

Comment (Cellebyte) [reply pkg/reconciler/layer2.go:72]: Ok :)

**Comment by watzkuh** [comment]
lgtm

**Pull request merged**